### PR TITLE
Update blog, email, and professionalhelp

### PR DESCRIPTION
**Pull request by merylke**
Added A11yVR, Equal Entry blog, Equal Entry newsletter, and Equal Entry professional services.

### CODE REVIEW

**Comment by ericwbailey** [comment]
Hi @merylke! I'm seeing some failing checks. Is that something you can take a look at, or is it something you'd like some help with? I think there's some indentation issues on line 1124.

**Comment by merylke** [comment]
@ericwbailey WOW! I copied the others to be sure this wouldn't happen. I found it.

It needs a { before "title," but can't find where to edit the file. How do I do that?

		"additional": "New York, United States",
			"url": "https://www.meetup.com/a11ynyc/"
		},
			"title": "A11yVR - Accessibility Virtual Reality",
			"desc

File: src/_data/resources.json [hunk 1]
```diff
@@ -1138,6 +1138,11 @@
 			"description": "Our group exists to advance accessibility and inclusive design. Our goal is to bring together NYC's accessibility community to share ideas, best practices, and our experiences.",
 			"additional": "New York, United States",
 			"url": "https://www.meetup.com/a11ynyc/"
+		},
+			"title": "A11yVR - Accessibility Virtual Reality",
```
Comment (ericwbailey) [inline src/_data/resources.json:1142]:
```suggestion
		},
		{
			"title": "A11yVR - Accessibility Virtual Reality",
```

**Review by ericwbailey — CHANGES_REQUESTED** [review]
I think if you accept this change it should do the trick!

**Review by merylke — COMMENTED** [review]
Looks good!

**Comment by merylke** [comment]
Don't see a way to submit. I did submit for review. It would not let me "approve."

File: src/_data/resources.json [hunk 2]
```diff
@@ -1138,6 +1138,11 @@
 			"description": "Our group exists to advance accessibility and inclusive design. Our goal is to bring together NYC's accessibility community to share ideas, best practices, and our experiences.",
 			"additional": "New York, United States",
 			"url": "https://www.meetup.com/a11ynyc/"
+		},
```
Comment (ericwbailey) [inline src/_data/resources.json:1141]:
```suggestion
		},
		{
```

File: src/_data/resources.json [hunk 3]
```diff
@@ -1310,6 +1315,11 @@
 			"description": "Get notified when we write new stuff",
 			"additional": "Axess Lab",
 			"url": "https://axesslab.com/alt-texts#thanks-for-making-the-web-better"
+		},
```
Comment (ericwbailey) [inline src/_data/resources.json:1318]:
```suggestion
		},
		{
```

**Review by ericwbailey — CHANGES_REQUESTED** [review]

**Review by ericwbailey — APPROVED** [review]
@merylke In the GitHub review process the person reviewing the proposed changes is in charge of accepting or requesting changes.

I snuck in another missing opening bracket, so we should be good to merge! Thank you for submitting this.

**Comment by merylke** [comment]
I'm such a Github dunce. I tried to close the pull request thinking it's all good ... but it said that it was not done yet? So sorry to make trouble!

**Comment by ericwbailey** [comment]
No trouble at all! Once the checks have cleared I'll merge this code and close the Pull Request. You did the hard part of submitting the PR 🙂

**Pull request merged**